### PR TITLE
Add new scripts to check if drawings need updates and code for new and improved fields

### PR DESCRIPTION
**Pull request by rminerptc**
Added 2 new scripts for checking if drawings need updates:
- detect-if-single-drawing-needs-update - will check one drawing to see if it needs an update
- detect-if-workspace-contains-drawings-needing-update - will check all the drawings in a workspace to see if any of them need an update

Added code to check the new isDangling field that was recently added to inspection symbols.

Added code to parse the new output string returned by the modify/status API, which contains clear results for modify actions.

Note that I will need to change launch.json to have production (cad.onshape.com) URLs instead of Demo-C URLs before this is merged.  I could not use production URLs now because the needed API changes are not on production yet.  So this will not be merged until 1.189 is released.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
             ],
             "program": "${workspaceFolder}/.compiledjs/create-note.js",
             "args": [
-                "--drawinguri=https://cad.onshape.com/documents/854e914aa52304e246a9eec9/w/abfdc05bfa1f41beddef6799/e/0bb00c2b35c0dad605592265",
```
Comment (yimaoatptc) [inline .vscode/launch.json]: It is good to define a variable for the server base.

Comment (rminerptc) [reply .vscode/launch.json]: Could you explain in a bit more detail what you mean?  I was following the pattern that Raj created in his onshape-ts-client repo - https://github.com/onshape-public/onshape-ts-client/blob/master/.vscode/launch.json

But I'm happy to improve this, once I understand what you mean.  Thanks.

**Review by tejan-onshape — APPROVED** [review]
Overall looks good.
however we will need to defer publishing changes for  modify/status API until 189 is release.

File: .vscode/launch.json (hunk 1 above)
Comment (rminerptc) [reply .vscode/launch.json]: I will leave this improvement to you.  Thanks.

**Pull request merged**